extensions/cdc-core/src/main/java/com/hazelcast/jet/cdc/ChangeEvent.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2020 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.cdc;
+
+import com.hazelcast.jet.annotation.EvolvingApi;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Information pertaining to a single data change event (insertion,
+ * delete or update), affecting either a single database record
+ * (in the case of relational databases) or document (in the case of
+ * NoSQL databases).
+ * <p>
+ * Each event has a <i>key</i>, which identifies the particular
+ * record or document being affected, and a <i>value</i>, which
+ * describes the actual change itself.
+ * <p>
+ * Most events have an <i>operation</i> associated with them which
+ * specifies the type of change being described (insertion, delete or
+ * update). This is really a property of the value, it's only replicated
+ * at this level for convenience. Only some special events, like
+ * heartbeats don't have an operation value.
```

Comment:
Could you please explain what is the `heartbeat` events ?

Reply:
If you set `heartbeat.interval.ms` in the Debezium connectors' config (not just the MySQL one, but in general), they will produce periodic heartbeat events. I haven't exposed this in our sources, but it's possible to enable (via `com.hazelcast.jet.cdc.impl.AbstractSourceBuilder.setCustomProperty`) and encounter them while using CDC via Jet. But is it worth a more than cursory mention in our Javadoc? Don't know. They get some mention, because they are the reason `Operation.UNSPECIFIED` exists... What would you like me to change?

Reply:
I see, I mean we can at least provide a link to it's definition and configuration ? I'm also fine if the explanation above present in Javadoc

Reply:
Ok. I don't want to explain too much, because I don't want to encourage anybody to use the heartbeats in Jet, but will add some clarifications.